app/src/web.ts

```diff
@@ -0,0 +1,40 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type { AppInfo, AppPlugin, AppLaunchUrl, AppState } from './definitions';
+
+export class AppWeb extends WebPlugin implements AppPlugin {
+  constructor() {
+    super({ name: 'App' });
+    if (typeof document !== 'undefined') {
+      document.addEventListener(
+        'visibilitychange',
+        this.handleVisibilityChange.bind(this),
+        false,
+      );
+    }
+  }
+
+  exitApp(): never {
+    throw this.unimplemented('Not implemented on web.');
+  }
+
+  async getInfo(): Promise<AppInfo> {
+    throw this.unimplemented('Not implemented on web.');
```

Comment:
I would suggest it will be a lot more useful (and developer-friendly) to return the information you can get from package.json, rather than throw an error.

Reply:
dwieeb already answered you on the other comment, the package.json is not accesible from the app/plugin, would need some CLI work first.

Reply:
Github is refusing to show me the other comment and his response, that's why I had to post it again.

I forgot this is a separate plugin now. Maybe Capacitor could make the app's package.json (or a subset of it) available to plugins.

Reply:
You have to click "Show resolved":
<img width="138" alt="image" src="https://user-images.githubusercontent.com/236501/97906685-eedf2280-1cf8-11eb-9b46-64800cbedb51.png">

Reply:
Thanks, the github review UI is getting better, but is still somewhat confusing at times.